pyscraper/sp/get-official-reports-new.py

```diff
@@ -92,8 +109,13 @@ def main():
         for i in range(1, int(options.historic, 10)):
             get_document_from_id(i)
 
+    elif options.start_range and options.end_range:
+        for i in range(int(options.start_range), int(options.end_range, 10)):
```

Comment:
Perhaps replace historic with this, seems unlikely we'll ever want to fetch all from 1 again and can use the range for this instead anyway.